src/Queue/JobList.js

```diff
@@ -1,84 +1,303 @@
+const baseUrl = 'https://nmpi.hbpneuromorphic.eu/api/v2/results/?collab_id=';
+const baseQueueUrl = 'https://nmpi.hbpneuromorphic.eu/api/v2/queue/?collab_id=';
+// url used to get collabs' ids 
+const baseGlobalUrl = "https://validation-v2.brainsimulation.eu";
+
+
+/*const useStyles = makeStyles((theme) => ({
+
+  spinIcon: { spin: false,
 
+  },
 
+}));
+*/
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      main: '#0b6623',
+    },
+    secondary: {
+      main: red[500],
+    },
+    running: {
+      main: yellow[500],
+    },
+  },
+});
 class JobList extends React.Component {
 
   constructor(props) {
+    
+    
     super(props)
     this.state = {
       jobs: [],
+      provJobList: [],
+      tagList:[], 
       error: '',
+      authToken: props.auth.token,
+      refreshState: false,
+      refreshDate :'',
+      collabList:[],
     }
+    
   }
 
-  componentDidMount(){
+  getTagsList = async()=> {
+    const tagsUrl = baseUrl + 'tags/?collab_id=' + this.props.collab;
+    const config = {headers: {'Authorization': 'Bearer ' + this.state.authToken}};
+    await axios.get(tagsUrl, config)
+        .then(res => {
+          console.log(res);
+          // this.setState({jobs: response.data.objects});
+          let availableTags = [];
+          res.data.objects.forEach(tag => {
+              console.log(tag.name);
+              availableTags.push(tag.name);
+              }
+          );
+          availableTags.sort();
+          console.log(availableTags);
+          this.setState({
+            tagList: availableTags.map(String)
+          });
+            }
+            )
+        .catch(error => {
+          console.log(error)
+          this.setState({errorMsg: 'Error retreiving data'})
+        })
+    
+        console.log('---taglist?---', this.tagList)
+}
+
+
+   getCollabList= async()=> {
+    const url = baseGlobalUrl + "/projects";
+    const config = {headers: {'Authorization': 'Bearer ' + this.state.authToken}};
+    await axios.get(url, config)
+        .then(res => {
+            let editableProjects = [];
+            res.data.forEach(proj => {
+                if (proj.permissions.UPDATE) {
+                    editableProjects.push(proj.project_id);
+                }
+            });
+            editableProjects.sort();
+            this.setState({
+              collabList: editableProjects.map(String)
+            });
+        })
+        .catch(err => {
+            console.log('Error: ', err.message);
+        });
+
+
+}
+
+  fetchData=async ()=>{
+
     let config = {
       headers: {
-        'Authorization': 'Bearer ' + this.props.auth.token,
+        'Authorization': 'Bearer ' + this.state.authToken,
       }
     }
-    axios.get(resultsUrl, config)
+    let resultsUrl = baseUrl+this.props.collab;
+    let queueUrl = baseQueueUrl +this.props.collab;
+
+
+    let currentdate = new Date();
+    let fetchDataDate = "Last updated: " + currentdate.getDate() + "/"
```

Comment:
I think we do date formatting in a couple of places, so I suggest we make this a function and move it to a `utils.js` file